<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="MacBook-Air-de-Diego.local" name="com.br.inmetrics.tests.android.TestMenuGherkin" tests="2" failures="1" timestamp="24 abr 2019 21:59:38 GMT" time="0.905" errors="0">
  <testcase name="setup" time="0.905" classname="com.br.inmetrics.tests.android.TestMenuGherkin">
    <failure type="java.lang.AssertionError" message="Setup error.">
      <![CDATA[java.lang.AssertionError: Setup error.
at org.testng.Assert.fail(Assert.java:85)
at com.br.inmetrics.frm.base.TestBase.setup(TestBase.java:32)
at com.br.inmetrics.tests.android.TestMenuGherkin.setup(TestMenuGherkin.java:676)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: com.br.inmetrics.frm.exceptions.GenericException: An generic exception occured.
at com.br.inmetrics.frm.controllers.AndroidController.setup(AndroidController.java:72)
at com.br.inmetrics.frm.base.DefaultBaseController.setup_(DefaultBaseController.java:173)
at com.br.inmetrics.frm.helpers.TestBaseHelper.doBefore(TestBaseHelper.java:129)
at com.br.inmetrics.frm.base.TestBase.setup(TestBase.java:28)
... 29 more
Caused by: java.lang.NullPointerException
at com.br.inmetrics.frm.controllers.AndroidController.setup(AndroidController.java:51)
... 32 more
]]>
    </failure>
  </testcase> <!-- setup -->
  <testcase name="CT0202_ValidarSenhasDoCartaoDoMenu" time="0.000" classname="com.br.inmetrics.tests.android.TestMenuGherkin">
    <skipped/>
  </testcase> <!-- CT0202_ValidarSenhasDoCartaoDoMenu -->
</testsuite> <!-- com.br.inmetrics.tests.android.TestMenuGherkin -->
